app/manifest/v2/_base.json

```diff
@@ -69,6 +69,7 @@
     "https://*.codefi.network/",
     "https://chainid.network/chains.json",
     "https://lattice.gridplus.io/*",
+    "https://metamask.github.io/*",
```

Comment:
Why is this necessary?

Usually we avoid any changes to the permissions

Reply:
I'm about to change it to a more restrictive URL, but we need it to fetch the list of allowed snaps from our registry.